crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -19,7 +19,7 @@ use starknet::signers::SigningKey;
 #[command(about = "Create an account with all important secrets")]
 pub struct Create {
     /// Account name under which account information is going to be saved
-    #[clap(short, long)]
+    #[clap(short, long, default_value = "", required_unless_present = "keystore")]
     pub name: String,
```

Comment:
not sure I understand this - so we do not want to write to our accounts file by default? I think that we should. We could have an account export (or something like this), to export the account to starkli-style json file. wdyt?